plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConnectorTest.java

```diff
@@ -192,6 +257,80 @@ public void testDropTable()
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    @Override
+    public void testRenameTable()
+    {
+        // Use CREATE TABLE instead of CREATE TABLE AS statement
```

Comment:
Can we fix this in base test? It seems we don't need CTAS to test rename table either?

Reply:
We could do it, but I hesitated to change in this PR because the test contains assertion for values.